e2e/src/step_definitions/media-viewer.ts

```diff
@@ -127,6 +127,32 @@ Then('I expect to see the document should be downloaded', async function () {
   await downloadPage.waitForDownloadToComplete(file);
 });
 
+When('the user selects the multimedia option', async () => {
+  await multimediaPage.clickMultimedia();
+  await genericMethods.sleep(5000);
+});
+
+When('the user selects play option', async () => {
+  await multimediaPage.clickPlayButton();
+});
+
+Then('I should see video should be in play mode', async function ()  {
+    await genericMethods.sleep(5000);
+    const screenshots = await browser.takeScreenshot();
+    this.attach(screenshots, 'image/png');
+});
+
+When('the user selects pause option', async () => {
+  await multimediaPage.clickPlayButton();
+});
+
+Then('I should see video should be in pause mode', async function () {
```

Comment:
again here, we're not really checking here, which means manual intervention is required to validate if the test has passed

Reply:
if ( pause) click success then video is in stop mode  and capturing the screenshot and that is attached to report .
else it throws error msg